src/components/ChipGroup/_chip-group.scss

```diff
@@ -1,14 +1,19 @@
 @import "vanilla-framework/scss/vanilla";
 
 .chip-group {
-  margin-top: -0.4rem;
-  padding-bottom: 0.5rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 1.25rem 0;
```

Comment:
the `chip-group` spacing `margin`/`padding` was overridden in different places, I got rid of the overrides and fixed the styling here.
| units | models |
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/36013798/195398050-2ee18b54-051c-4ef7-b615-cebff5a70e2f.png) | ![image](https://user-images.githubusercontent.com/36013798/195398078-6b77d54b-3753-4788-ba38-9b0621e7344b.png)